src/patternfly/components/OverflowMenu/docs/overflow-menu-group-types.md

```diff
@@ -0,0 +1,8 @@
+The action group consists a primary and secondary action. Any additional actions are part of the overflow control dropdown.
+
+### Overflow menu group types
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-overflow-menu__group` | `<div>` | Initiates the overflow menu component group. |
+| `.pf-m-button-group` | `.pf-c-overflow-menu__group` | Modifies overflow menu group spacing. Spacer value is set to `var(--pf-c-overflow-menu__group--m-button-group--spacer)`. Child spacer value is set to `var(--pf-c-overflow-menu__group--m-button-group--space-items)`. |
+| `.pf-m-icon-button-group` | `.pf-c-overflow-menu__group` | Modifies overflow menu group spacing. Spacer value is set to `var(--pf-c-overflow-menu__group--m-icon-button-group--spacer)`. Child spacer value is set to `var(--pf-c-overflow-menu__group--m-icon-button-group--space-items)`. |
```

Comment:
I think you could clarify here that when you have an "icon button" you should use this modifier and is it an icon inside a button using the `pf-m-plain` modifier?